article/structure.py

```diff
@@ -23,7 +23,7 @@ def total_reading_time(self):
 
 
 PERSONA_NEW_ARTICLES = ArticleGroup(
-    key='persona-new',
+    key='persona_new',
```

Comment:
Maybe just “name” instead of “key”? Bit confusing as this not a key-value struct